backend/api/Controllers/ParcelController.cs

```diff
@@ -56,9 +57,49 @@ public ParcelController(ILogger<ParcelController> logger, IConfiguration configu
         /// <returns></returns>
         [HttpGet]
         [HasPermission(Permissions.PropertyView)]
-        public IActionResult GetMyParcels(double neLat, double neLong, double swLat, double swLong, int? agencyId = null, int? propertyClassificationId = null)
+        public IActionResult GetParcels(double? neLat = null, double? neLong = null, double? swLat = null, double? swLong = null)
         {
-            var parcels = _pimsService.Parcel.GetNoTracking(neLat, neLong, swLat, swLong, agencyId, propertyClassificationId);
+            if (neLat == null) throw new BadRequestException($"Query parameter {nameof(neLat)} required.");
+            if (neLong == null) throw new BadRequestException($"Query parameter {nameof(neLong)} required.");
+            if (swLat == null) throw new BadRequestException($"Query parameter {nameof(swLat)} required.");
+            if (swLong == null) throw new BadRequestException($"Query parameter {nameof(swLong)} required.");
+
+            var parcels = _pimsService.Parcel.GetNoTracking(neLat.Value, neLong.Value, swLat.Value, swLong.Value);
+            return new JsonResult(_mapper.Map<Model.Parts.ParcelModel[]>(parcels));
+        }
+
+        /// <summary>
+        /// Get all the parcels that satisfy the filter parameters.
+        /// </summary>
+        /// <param name="neLat"></param>
+        /// <param name="neLong"></param>
+        /// <param name="swLat"></param>
+        /// <param name="swLong"></param>
+        /// <returns></returns>
+        [HttpGet("filter")]
+        [HasPermission(Permissions.PropertyView)]
+        public IActionResult GetParcelsWithFilter()
```

Comment:
New /api/parcels/filter endpoint

Reply:
I wouldn't want to begin a "pure" REST argument here but would like to suggest we keep a single endpoint `/api/parcels` that returns a result based on the filter parameters provided. If no filters provided, you get all (with pagination of course, not the whole DB). If you supply filters, you get less properties - only those matching the filters

Reply:
https://specs.openstack.org/openstack/api-wg/guidelines/pagination_filter_sort.html#filtering
https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/